### ISSUE TITLE
THIS FORK IS NOT MAINTAINED - READ BEFORE SUBMITTING

### ISSUE THREAD

**Issue by leonadler** [Ericsson/exchangecalendar#610]
*TL;DR:* **new fork** -> [ExchangeCalendar/exchangecalendar](https://github.com/ExchangeCalendar/exchangecalendar/)
Download the [latest release](https://github.com/ExchangeCalendar/exchangecalendar/releases) and check if your issue still exists before submitting it there.

---

Since Thunderbird 52, the plugin provided by this repository stopped working reliably.
Since the fork by Ericsson seems not to be maintained anymore and no pull requests are accepted, a fork which fixes most issues with TB 52 can be found at [ExchangeCalendar/exchangecalendar](https://github.com/ExchangeCalendar/exchangecalendar/).

Before copying your issue to that fork, please try if [the most recent release](https://github.com/ExchangeCalendar/exchangecalendar/releases) in that fork resolves your issue.

Thanks! :love_letter:

**Comment by berni2288** [comment]
Thanks Leon, you are the best!

**Comment by anhdy41** [comment]
Confirmed.  The 4.0 beta on the referred to link above did fix nearly all of my issues.  Thank you to everyone who is keeping this project going.

**Comment by ghost** [comment]
I upgraded Exchange EWS Provider to 4.0.0-beta4 but I still experience the same problem : tasks with dates are only displayed in the calendar view when they are created; once you change view (week for instance), they disappear ...

**Comment by ghost** [comment]
Just to precise things: I have to click on "Save and reset" in "Exchange Properties (EWS)" to make them appear again but when I switch to another week in the calendar view they disappear agin.

**Comment by leonadler** [comment]
@bnysten don't hijack the issue, this is not the place to report bugs. Please report the bug with the appropriate logs in the [maintained fork](https://github.com/ExchangeCalendar/exchangecalendar/).

**Comment by Kachengo** [comment]
"Thank you to everyone who is keeping this project going."

Yes! Vielen Dank!
